Libraries/Interface/logo.h

```diff
@@ -0,0 +1,28 @@
+#include <cstring>
+#include <vector>
+
+/**
+ * Class of the giant snoke logo
+ * @prop {int}                     x           - x coordinate of left logo side
+ * @prop {int}                     y           - y coordinate of top logo side
+ * @prop {PointStyle}              style       - style of the cells with logo
+ * @prop {PointStyle}              shadowStyle - style of the cells with shadow of the logo
+ * @prop {short}                   language    - language of the inscription at the logo, 0 - en, 1 - ru
+ * @prop {map<short, const char*>} inscription - dictionary with translations for inscription
+ */
+class Logo
+{
+    private:
+        int x, y;
+        PointStyle style, shadowStyle;
+        short language;
+        std::vector<std::vector<const char*>> inscription;
+
+    public:
+        void draw();
+        void setPosition(int x, int y);
+        Point getSize();
+        Logo(int x, int y, PointStyle style, PointStyle shadowStyle, int language);
+};
+
+#include "logo.cpp"
```

Comment:
Avoid adding the *.cpp files in the end of the headers
Rewrite the *.cpp and *.h files to be Makefile compatitable(from the Game branch)
To achieve this make all includes needed for the function in the header, add preprocessor commands 
#ifndef, #define, {header code}, #endif
And start *.cpp files with an addition of a corresponding header